app/controllers/handovers_controller.rb

```diff
@@ -5,24 +5,7 @@ class HandoversController < PrisonsApplicationController
 
   layout 'handovers'
 
-  before_action :check_prerequisites_and_prepare_variables, except: :index
-  before_action :ensure_spo_user, only: :index
```

Comment:
Do we definitely want to remove the `ensure_spo_user` check?

Reply:
ensure_spo_user check was only being used for the old index page which is now deleted :+1: